concrete/src/Support/Symbol/ClassSymbol/ClassSymbol.php

```diff
@@ -56,8 +70,16 @@ class ClassSymbol
     public function __construct($alias, $fqn, $facade = null)
     {
         $this->reflectionClass = new ReflectionClass($fqn);
-        $this->fqn = $fqn;
-        $this->alias = $alias;
+        $this->fqn = ltrim($fqn, '\\');
+        $this->alias = ltrim($alias, '/');
+        $p = strrpos($this->alias, '\\');
```

Comment:
Could this be replaced with:

```php
<?php
$exploded = explode('\\', $fqn);
$this->aliasBasename = array_pop($exploded);
$this->aliasNamespace = implode('\\', $fqn);
```
?
IF so that'd be easier to read IMO

Reply:
Sure! Tomorrow I'll update this PR

Reply:
PR updated